app/players/player.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    @abstractmethod
+    def __init__(self,
+                 nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
```

Comment:
The return type hint should be int, not None. According to the requirements, all concrete implementations of get_rating must return an integer value. The abstract method's signature should reflect the expected return type of its implementations.